CMakeLists.txt

```diff
@@ -272,7 +272,6 @@ if(ENABLE_BEST)
   cvc4_set_option(USE_CLN ON)
   cvc4_set_option(USE_CRYPTOMINISAT ON)
   cvc4_set_option(USE_GLPK ON)
```

Comment:
Let's use `USE_EDITLINE` in out "best" configuration, too.

Reply:
Done!